src/mbgl/gl/renderer_backend.cpp

```diff
@@ -64,5 +67,72 @@ void RendererBackend::setScissorTest(bool enabled) {
 
 RendererBackend::~RendererBackend() = default;
 
+void RendererBackend::initShaders(gfx::ShaderRegistry& shaders) {
+    constexpr auto shaderName = "background_generic";
+
+    gl::Context& glContext = static_cast<gl::Context&>(*context);
+
+//    "layout (location = 0) in vec2 a_pos;\nuniform mat4 u_matrix;\n\nvoid main() {\n
+    // gl_Position = u_matrix * vec4(a_pos, 0, 1);\n}\n", fragment = "uniform vec4 u_color;\nuniform float u_opacity;\n\nvoid main() {\n    fragColor = u_color * u_opacity;\n\n#ifdef OVERDRAW_INSPECTOR\n    fragColor = vec4(1.0);\n#endif\n}\n"
+    
+    auto shader = shaders.get<gl::ShaderProgramGL>(shaderName);
+    if (!shader) {
+        constexpr auto vert = R"(
```

Comment:
On more picky compilers (like my windows desktop), `#version 300 es` must literally be the first thing, it even complains on the line break here.